src/ansys/aedt/core/generic/configurations.py

```diff
@@ -2319,24 +2319,24 @@ def export_config(self, config_file=None, overwrite=False):
 
         if "" in pin_mapping:
             del pin_mapping[""]
-        for k, l in pin_mapping.items():
+        for key, value in pin_mapping.items():
```

Comment:
```suggestion
        for key, values in pin_mapping.items():
```
Since you are going to iterate over "value", could you use a plural form so that we know it's a collection of things ?
Please, propagate that to the following lines and use "value" instead of "val"